azurerm/resource_arm_subnet.go

```diff
@@ -161,6 +166,11 @@ func resourceArmSubnetCreateUpdate(d *schema.ResourceData, meta interface{}) err
 		AddressPrefix: &addressPrefix,
 	}
 
+	if v, ok := d.GetOk("private_link_service_network_policies"); ok {
+		privateLinkServiceNetworkPolicies := v.(string)
+		properties.PrivateLinkServiceNetworkPolicies = &privateLinkServiceNetworkPolicies
```

Comment:
We can merge these two lines here
```suggestion
		properties.PrivateLinkServiceNetworkPolicies = utils.string(v.(string))
```

Reply:
No, because the value in Azure is a string :/